vininfo/dicts/wmi.py

```diff
@@ -8,36 +8,74 @@
     '000': 'Maserati',
     '04W': 'Buick',
     '0VF': 'Ford',
+    '10T': 'Oshkosh',
     '112': 'Volkswagen',
     '115': 'Mercedes-Benz',
     '117': 'Volkswagen',
     '119': 'Replica/Kit Makes',
+    '11V': 'Ottawa',
     '123': 'Mercedes-Benz',
     '124': 'Chevrolet',
     '137': 'AM',
+    '145': 'Hyundai',
+    '15G': 'Gillig',
+    '177': 'Thomas',
     '178': 'Jaguar',
+    '17N': 'John Deere',
     '17V': 'Ford',
+    '18X': 'WRV',
     '19': 'Acura',
+    '19U': 'Acura',
     '19X': 'Honda',
     '1A4': 'Chrysler',
     '1A8': 'Chrysler',
+    '1AC': 'AMC',
+    '1AM': 'AMC',
     '1B': 'Dodge',
+    '1B3': 'Dodge',
+    '1B4': 'Dodge',
+    '1B6': 'Dodge',
+    '1B7': 'Dodge',
+    '1BA': 'Blue Bird',
+    '1BB': 'Blue Bird',
+    '1BD': 'Blue Bird',
     '1C': 'Chrysler',
+    '1C3': 'Chrysler',
     '1C4': 'Dodge',
     '1C6': 'Ram',
     '1C8': 'Chrysler',
+    '1C9': 'Chance',
     '1CN': 'Lincoln',
+    '1CY': 'Crane Carrier',
     '1D': 'Dodge',
+    '1D3': 'Dodge',
```

Comment:
Two letter dict keys already decribe implicitly three letters varians. E.g. 1D includes 1D3, 1D4 etc.
That's done to keep the dict minimized. So those need to be dropped.
If unsure you can always check corner cases in tests.

Reply:
I agree, but I wonder if it's not a good idea to remove all the two-character keys? Because in reality the manufacturer is always identified by three characters, here the two character key are used as an easy fallback but in reality this will not give a correct in all cases.

```
    @property
    def wmi(self) -> str:
        """WMI (World Manufacturers Identification)"""
        return self.num[:3]

    @property
    def brand(self) -> Brand:
        """Brand object."""

        wmi = self.wmi

        brand = WMI.get(wmi)

        if not brand:
            brand = WMI.get(wmi[:2])
 ```

Reply:
> but in reality this will not give a correct in all cases.

Let's say it is considered correct till it's not proved otherwise %P
So let's keep a compacted version until we have issues reported.